java-client/session-examples/build.gradle

```diff
@@ -43,6 +43,7 @@ applicationDistribution.into('bin') {
     from(createApplication('execute-script', 'io.deephaven.client.examples.ExecuteScript'))
     from(createApplication('execute-code', 'io.deephaven.client.examples.ExecuteCode'))
     from(createApplication('subscribe-fields', 'io.deephaven.client.examples.SubscribeToFields'))
+    from(createApplication('connect-check', 'io.deephaven.client.examples.ConnectCheck'))
```

Comment:
I'd vote split this into two PRs, unless this is somehow related to testing the other change(s)? it doesnt look like it could be though, as those are basically just variable names, or the hostname that envoy uses to see the other docker container.